v2/samples/direct_method_invocation_sample/deployment.json

```diff
@@ -0,0 +1,68 @@
+﻿{
+  "moduleContent": {
+    "$edgeAgent": {
+      "properties.desired": {
+        "schemaVersion": "1.0",
+        "runtime": {
+          "type": "docker",
+          "settings": {
+            "minDockerVersion": "v1.25",
+            "loggingOptions": ""
+          }
+        },
+        "systemModules": {
+          "edgeAgent": {
+            "type": "docker",
+            "settings": {
+              "image": "microsoft/azureiotedge-agent:1.0-preview",
+              "createOptions": ""
+            }
+          },
+          "edgeHub": {
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "settings": {
+              "image": "microsoft/azureiotedge-hub:1.0-preview",
+              "createOptions": ""
+            }
+          }
+        },
+        "modules": {
+          "engineAnalyzer": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "settings": {
+              "image": "edgetest.azurecr.io/engineanayzer:varun2",
```

Comment:
Can we have this image be hosted in Docker hub? Or alternately, put some placeholder image name here that users are expected to replace with an actual image reference.

Reply:
Ah, thanks for catching that. That was an oversight.. meant to use a placeholder there. Will fix.